src/TConsole.cpp

```diff
@@ -366,8 +368,144 @@ std::tuple<std::string, std::vector<std::string>> TConsole::ParseCommand(const s
     return { Command, Args };
 }
 
+template <class... Ts>
+struct overloaded : Ts... {
+    using Ts::operator()...;
+};
+template <class... Ts>
+overloaded(Ts...) -> overloaded<Ts...>;
+
 void TConsole::Command_Settings(const std::string&, const std::vector<std::string>& args) {
-    if (!EnsureArgsCount(args, 1, 2)) {
+
+    static constexpr const char* sHelpString = R"(
+    Settings:
+        settings help                               displays this help
+        settings list                               lists all settings
+        settings get <category> <setting>           prints current value of specified setting
+        settings set <categoty> <setting> <value>   sets specified setting to value
+        )";
+
+    if (args.size() == 0) {
```

Comment:
use `EnsureArgsCount` here for consistency if possible, not sure rn if it has an overload for this use case, but im pretty sure it has a min+max variant